pkg/controller/atlasproject/project_test.go

```diff
@@ -147,10 +141,16 @@ func TestHandleProject(t *testing.T) {
 				mockPeeringEndpointAPI.EXPECT().
 					ListPeeringConnectionsExecute(admin.ListPeeringConnectionsApiRequest{ApiService: mockPeeringEndpointAPI}).
 					Return(&admin.PaginatedContainerPeer{}, nil, nil)
+				mockTeamApi := mockadmin.NewTeamsApi(t)
```

Comment:
If the translation layer is implemented, these tests mocks tend to get easier as the test here only needs to cover up to the translation layer call, not up to the SDK call.

Reply:
Yes agreed, the translation layer will be implemented as a part of a following PR!